dask/dataframe/groupby.py

```diff
@@ -413,6 +421,17 @@ def __getattr__(self, key):
         except KeyError as e:
             raise AttributeError(e)
 
+    def fillna(self, value):
+        meta = self.obj.head(1).groupby(self.index).fillna(value)
+        if isinstance(meta, pd.DataFrame) and self._slice is not None:
+            meta = meta[self._slice]
+        columns = meta.columns if isinstance(meta, pd.DataFrame) else meta.name
+
+        result = map_partitions(_fillna, self.obj, value, self.index, columns,
+                                meta=meta, token='fillna')
```

Comment:
This came up here: http://stackoverflow.com/questions/38901845/basic-groupby-operations-in-dask
So it might have value in it although it would be a lot more helpful if dask supported the `method` parameter.

I'll try adding suport for it.